packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -158,13 +158,39 @@ class Solver {
   /// Attempts to add the constraints in the list to the solver. If it cannot
   /// add any for some reason, a cleanup is attempted so that either all
   /// constraints will be added or none.
+  ///
+  /// Check the [Result] returned to make sure the operation succeeded. Any
+  /// errors will be reported via the `message` property on the [Result].
+  ///
+  /// Possible [Result]s:
```

Comment:
here and elsewhere:

when you have a bulleted list, you need a blank line before the first list item, otherwise the list becomes one long paragraph.